src/spaceone/inventory/manager/workload/job_manager.py

```diff
@@ -0,0 +1,178 @@
+import logging
+
+from spaceone.inventory.connector.workload.job import JobConnector
+from spaceone.inventory.libs.manager import KubernetesManager
+from spaceone.inventory.model.workload.job.cloud_service import JobResponse, JobResource
+from spaceone.inventory.model.workload.job.cloud_service_type import CLOUD_SERVICE_TYPES
+from spaceone.inventory.model.workload.job.data import Job
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class JobManager(KubernetesManager):
+    connector_name = "JobConnector"
+    cloud_service_types = CLOUD_SERVICE_TYPES
+
+    def collect_cloud_service(self, params):
+        _LOGGER.debug("** Job Start **")
+        """
+        Args:
+            params:
+                - options
+                - schema
+                - secret_data
+                - filter
+                - zones
+        Response:
+            CloudServiceResponse
+        """
+        collected_cloud_services = []
+        error_responses = []
+
+        secret_data = params["secret_data"]
+        pod_name = ""
+
+        ##################################
+        # 0. Gather All Related Resources
+        # List all information through connector
+        ##################################
+        pod_conn: JobConnector = self.locator.get_connector(
+            self.connector_name, **params
+        )
```

Comment:
Because only one `self.connector_name` is used
It looks better to write directly to the relevant part without using it as a class variable.

Also, the `get_connector()` method can receive an object as a parameter.
So it would be better to change like below.
```python
pod_conn: JobConnector = self.locator.get_connector(
            JobConnector, **params
        )
```